### ISSUE TITLE
Subset overlay confusion

### ISSUE THREAD

**Issue by blelump** [the-human-colossus-foundation/oca-spec#46]
### Problem overview 

As defined in the spec, the subset overlay creates a subset over a set of attributes that the given capture base defines. Working on a subset is convenient in the presentation layer and post-processing. The main problem with a subset is that it is unknown when actually to apply it. It lacks the intent of use.

The novel concept of the presentation layer takes the responsibility of the subset in the case of presentation but does not in the remaining cases. However, due to lacking intent of use and therefore, limited applicability, what's the DSWG opinion about the existence of this particular overlay?

**Comment by pknowl** [comment]
Yes, shifting responsibility for subsets to the presentation layer makes sense. The concept of schema subsets was initially intended to ensure that exact fields could be extracted from different data sources for aggregation after the harmonisation process. If the presentation layer can cover that requirement, I’m happy to bin this overlay type.
